pkgs/applications/networking/cluster/tgswitch/default.nix

```diff
@@ -10,7 +10,11 @@ buildGoPackage rec {
     sha256 = "sha256-6hErfI7LEJFgOoJR8IF9jTSBwqbQYeGiwdeJShqxVQ0=";
   };
 
-  goPackagePath = "github.com/warrensbox/tgswitch";
+  vendorSha256 = null;
+
+  ldflags = [ "-s" "-w" ];
```

Comment:
Can you add comments about these ldflags ? I guess `-s` is for stripping the symbols. not sure about `-w`.

Reply:
These are common flags used in nixpkgs for smaller binary size. See https://pkg.go.dev/cmd/link.

Reply:
Ah, got it. Thanks!